slurmpilot/slurm_wrapper.py

```diff
@@ -104,6 +104,10 @@ def schedule_job(
         job_info.check_path()
         cluster_connection = self.connections[job_info.cluster]
         home_dir = self.home_dir[job_info.cluster]
+
+        root_dir = job_info.remote_dir if job_info.remote_dir is not None else home_dir
+        root_path = os.path.join(root_dir, "slurmpilot")
+
```

Comment:
Thanks for the catch, intially this option was supported through configs. To fix it, we would need to do this instead:
```suggestion
        root_dir = self.config.remote_slurmpilot_path(job_info.cluster)
```
which would allow to define in your config the location to use per cluster (which makes sense since I suppose it will stay constant).

Alternatively, we could also allow to specify it per job as suggested here. Perhaps we could support both still, by doing:

```python
        if job_info.remote_dir is None:
            root_dir = self.config.remote_slurmpilot_path(job_info.cluster)
        else:
            root_dir = job_info.remote_dir
```

I have a preference for the second option since it is documented, we can also merge your change and I can make it compatible with config later.

Reply:
I see the point of defining the location in the config. However, we do have the practice of having different workspaces for different projects, so it would be useful to give the user the ability to specify it per job as well.

I'd prefer the following:
```
if remote path specified for the job:
    use that
elif remote path specified in config for the cluster:
    use that
else:
    use home directory
```

Reply:
Yes, I agree but it is the same behavior as the snippet I gave above right?

Reply:
Ah, I see that it is!

I've pushed a commit with this change.